src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsalertsapi/model/Alert.kt

```diff
@@ -113,4 +113,23 @@ data class Alert(
     example = "Firstname Lastname",
   )
   val lastModifiedByDisplayName: String?,
+
+  @Schema(
+    description = "The date and time the alert's active duration end date is last set.",
```

Comment:
Maybe reword these similar to:

- "The date and time the alert was last set to expire"
- "The username of the user who the alert was last set to expire"
- "The displayable name of the user who the alert was last set to expire"